index.js

```diff
@@ -1,38 +1,16 @@
-function RandomCat() { }
-
-RandomCat.prototype.get = function (options) {
-  options = options || {};
-
-  if (options.dummyText) {
-    options.dummyText = escape(options.dummyText);
-  }
-
-  var randomSizes = [50, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600];
-
-  var urlParts = [
-      options.gray ? 'g' : false
-    , options.width || randomSizes[Math.ceil(Math.random() * 12)]
-    , options.height || randomSizes[Math.ceil(Math.random() * 12)]
-    , options.category || 'cats'
```

Comment:
I believe you've lost the category and default here...'cats' being the default is pretty important

Reply:
okay

Reply:
since this is an exclusive random  cat category ... why do we need to give category option?

Reply:
It's not a good thing to change what options you can pass.  If someone had been using "Dogs" or anything else, it would break that.

You are right though, this is a random cat lib, so why give another option?  I think in this case it is the principal of backward compatibility.